### ISSUE TITLE
Problem converting ONNX model to TensorRT 7.2.1 (worked in previous 7.1.3 release)

### ISSUE THREAD

**Issue by FamousDirector** [onnx/onnx-tensorrt#566]
With the following command:
`./onnx2trt /project/model_repository/osnet_ain_fused_engine/1/osnet_ain_fused.onnx -o /project/model_repository/osnet_ain_fused_engine/1/osnet_ain_fused.engine -b 128`

I get the following error:
```
[2020-11-12 00:24:53   ERROR] (Unnamed Layer* 512) [Shuffle]: at most one dimension may be inferred
While parsing node number 494 [BatchNormalization -> "1049"]:
ERROR: /opt/tensorrt/TensorRT/parsers/onnx/onnx2trt_utils.cpp:1517 In function scaleHelper:
[8] Assertion failed: dims.nbDims == 4 || dims.nbDims == 5
```
Here is a [link](https://drive.google.com/file/d/1dcZ8Yo11jgPRRxv9_nImX_Xu3MJr0C6E/view) to the model I'm trying to convert.
Please advise if I can provide anymore information. This conversion worked fine in previous releases.

**Comment by phamdat09** [comment]
I have the same problem as it !! @FamousDirector Did you fix it ? Thanks

**Comment by FamousDirector** [comment]
No I still have the issue.

**Comment by phamdat09** [comment]
I figure out that the problems come from the function `torch.BatchNorm1d` ( I convert pytorch -> onnx -> tensorrt).

**Comment by pranavm-nvidia** [comment]
[This PR](https://github.com/onnx/onnx-tensorrt/pull/569) should fix it.

**Comment by kevinch-nv** [comment]
@FamousDirector @phamdat09 can you confirm that @pranavm-nvidia's PR fixed your issue?

**Comment by FamousDirector** [comment]
@kevinch-nv Yes this fix did the job.

**Comment by mika-fischer** [comment]
When will this be fixed in TensorRT?

**Comment by pranavm-nvidia** [comment]
@mika-fischer It's already fixed. The ONNX parser [submodule commit in TensorRT](https://github.com/NVIDIA/TensorRT/tree/master/parsers) includes my PR.

**Comment by mika-fischer** [comment]
Well I'm using TensorRT 7.2.3.4, and still have this issue. I'm using the releases from nVidia, not the OSS version. As I understand it, the OSS version is not a replacement for the official release, right?

Is it recommended to use the parser from the OSS version with the inference engine from nVidia? Then why include the parsers in the official release at all?

In any case the question is, when will nVidia release a fixed version of TensorRT?

**Comment by pranavm-nvidia** [comment]
> As I understand it, the OSS version is not a replacement for the official release, right? 

Yes, OSS is not a full replacement for the official releases, since only the parsers, plugins, samples, and python bindings are open-source. 

> Is it recommended to use the parser from the OSS version with the inference engine from nVidia?

The official release is built with a specific commit of the parser, whereas OSS is constantly being updated. So OSS will often include commits not present in the official releases. That's pretty much unavoidable at this point since it's not practical for us to make an official release for every commit that goes into OSS. 

If you need a fix from OSS, you can either build from source or wait for the next official release of TensorRT
(if you don't want to wait, I'd recommend building). 

> Then why include the parsers in the official release at all?

IMO it's still good to include the parsers in the official release so that users aren't forced to build from source. 

> In any case the question is, when will nVidia release a fixed version of TensorRT?

The next release, which should be sometime next month, will include this fix.

**Comment by mika-fischer** [comment]
All right, thanks for clearing things up for me and the prompt response!

**Comment by ttanzhiqiang** [comment]
https://github.com/ttanzhiqiang/onnx_tensorrt_project
